test/fiddle/test_pointer.rb

```diff
@@ -285,7 +285,7 @@ def test_size
       end
       assert_equal 0, Pointer.new(0).size
       assert_equal 0, Pointer.new(0).ref.size
```

Comment:
Let's not omitting parenthesis for all assertions:

```suggestion
      assert_equal(0, Pointer.new(0).size)
      assert_equal(0, Pointer.new(0).ref.size)
```